pkgs/applications/kde/kpublictransport.nix

```
@@ -0,0 +1,13 @@
+{
+  mkDerivation, lib,
+  extra-cmake-modules,
+}:
+
+mkDerivation {
+  pname = "kpublictransport";
+  meta = {
+    license = [ lib.licenses.cc0 ];
+    maintainers = [ lib.maintainers.samueldr ];
+  };
```

```suggestion
  meta = with lib; {
    license = [ licenses.cc0 ];
    maintainers = [ maintainers.samueldr ];
  };
```
Also meta should be last.